File: 240914/함수를 이용한 소수 판별/decimal-decisions-using-functions.c
#include <stdio.h>
#include <stdbool.h>
#include <math.h>

bool is_prime(int n) {
    int i;
    if (n<2) return false;
    for (i=2; i<=sqrt(n); i++) {
        if (n%i==0) return false;
    }
    return true;
    
}
int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    
    int sumN = 0;
    int i;
    for (i=a; i<=b; i++) {
        if (is_prime(i)) {
            sumN += i;
        }
    }
    printf("%d", sumN);
    return 0;
}

File: 240914/함수를 이용한 소수 판별/decimal-decisions-using-functions.py
import math


def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(math.sqrt(n))+ 1):
        if n % i == 0:
            return False
    return True


a, b = map(int, input().split())
sumN = 0
for i in range(a, b+1):
    if is_prime(i):
        sumN += i

print(sumN)